src/components/ArtifactCard.vue

```diff
@@ -0,0 +1,232 @@
+<template>
+  <p-card class="artifact-card" :class="classes.root">
+    <div class="artifact-card__body" :class="classes.body">
+      <header class="artifact-card__header" :class="classes.header">
+        <h6 class="artifact-card__subheader" :class="classes.subheader">
+          {{ artifact.type }}
+        </h6>
+        <h3 v-if="hasKey">
+          {{ artifact.key }}
+        </h3>
```

Comment:
I don't know what this should be but an `h6` followed by an `h3` seems a little off to me? I thought generally headings were supposed to be in order semantically. Could be way off and this is fine.

Reply:
You know I can't find anything about that but I'm also not confident this is semantically helpful - I've dumped both